iceberg/iceberg-handler/src/test/results/positive/alter_table_to_iceberg.q.out

```diff
@@ -103,6 +103,7 @@ Table Parameters:
 	metadata_location   	hdfs://### HDFS PATH ###
 	numFiles            	1                   
 	numRows             	5                   
+	orc.files.only      	false               
```

Comment:
I'm not sure if I get the intention of orc.files.only property, but tbl_orc clearly looks like an orc table, isn't it the case? I can see that it has no partitions (so I cannot imagine how can we store other data in other formats here)
so if orc.files.only evaluates to false, we'll miss vectorization opportunity, right?

Reply:
Sorted